.github/workflows/build.yml

```diff
@@ -134,6 +134,14 @@ jobs:
           registry: 'ghcr.io/wayblueorg'
           pubkey: 'https://raw.githubusercontent.com/wayblueorg/wayblue/live/cosign.pub'
 
+      - name: Verify base image
+        if: ${{ contains(env.IMAGE_NAME, 'cosmic') }}
+        uses: EyeCantCU/cosign-action/verify@v0.3.0
+        with:
+          containers: ${{ env.BASE_IMAGE_NAME }}:${{ env.IMAGE_MAJOR_VERSION }}
+          registry: 'ghcr.io/wayblueorg'
+          pubkey: 'https://raw.githubusercontent.com/ublue-os/cosmic/main/cosign.pub'
```

Comment:
check the keys, this is the same pubkey as ublue-os/main, so this won't fix the problem. The problem is something else